src/repost.rs

```diff
@@ -5,7 +5,7 @@ use near_sdk::{env, AccountId, Promise};
 
 
 fn repost_internal(post: Post, contract_address: AccountId) -> near_sdk::serde_json::Value {
-    let post_link = format!("https://near.social/#/devgovgigs.near/widget/Post?id={}", post.id);
+    let post_link = format!("/#/devgovgigs.near/widget/Post?id={}", post.id);
```

Comment:
"#" is not required anymore, let's drop it

```suggestion
    let post_link = format!("/devgovgigs.near/widget/Post?id={}", post.id);
```